eth2.0-implementers-calls/call_021.md

```diff
@@ -0,0 +1,654 @@
+
+In regards to the client, comfortable to start doing Block production. Once we finish our update to v0.8, we'll start doing our testnet. Then getting it to work in-browser.
+
+Going back onto assembly script.
+
+Also, NIM, we're coming after you guys on the ERC20 contract, we'll get ya.
+
+#### 3.2.9 [Parity](https://github.com/paritytech/parity-ethereum)
+
+**Timestamp: [27:54](https://youtu.be/YB8o_5qjNBc?t=1624)**
+
+**Wei Tang**: Finished the markolization library last week, hopefully, want to extend that into a caching library, but there are still a few missing pieces.
+
+We're trying to update to v0.8 spec. The issue we had was in the SSE, a substantial change for us. Large refactoring of the codebase.
+
+### 3.3 [Research Updates](https://youtu.be/YB8o_5qjNBc?t=1725)
+
+**Timestamp: [29:08](https://youtu.be/YB8o_5qjNBc)**
+
+#### 3.3.1 [Phase 0](https://docs.ethhub.io/ethereum-roadmap/ethereum-2.0/eth-2.0-phases/)
+
+**Timestamp: [34:50](https://youtu.be/YB8o_5qjNBc?t=2083)**
+
+**Justin Drake**: In parallel to Phase 1 & Phase 2, the research team is doing is more education about Phase 0. Educational documents have been made.
+
+On July 15th 1:00 PM GMT there will be a 2nd Ethereum 2.0 AMA, an opportunity to ask questions related to your implementation work.
+
+Now that the spec is frozen, feel free to reach out with questions about the design. I'm Justin Drake on Telegram.
+
+
+#### 3.3.2 [Phase 1](https://docs.ethhub.io/ethereum-roadmap/ethereum-2.0/eth-2.0-phases/)
+
+**Timestamp: [29:08](https://youtu.be/YB8o_5qjNBc)**
+
+**Vitalik Buterin**: On research-side there's a list of To-Do's for Proof-of-Custody and Shard Blocks.
+
+Discovered our approach to light clients, involving Fiat Shimmering and Active Experts, didn't make sense.
+
+Found a simpler approach: Sign over the committee's route and allow clients to verify the committee's route directly. There's still some decisions to what committees are for each slot and how those committees change.
+
+In general:
+  * Phase 0, including in the blocks a route of the crosslink committees should be fine
+  * Phase 1, including a route of the persistent committees
+
+We get nice, really efficient, light clients. There's still just parameters to decide.
+
+Figuring out the exact structure of data in crosslinks.
+
+Not too much to do on the Shard Block side.
+
+Have a PR which switches from multiple attestations and a proposers signature to just having one signature that includes a proposer in the attestations. That's an option as well.
+
+Not seeing any unexpected difficulties on the Phase 1 side. It's looking better and better.
+
+**Justin Drake**: To add on Phase 1, we're likely to have some cryptanalysis done.  We might have Kovatovich analyze the cryptography used there.
+
+One of the primitives we're using is the Lujon symbol, as a PRF. It would still be good to have it audited because it's an assumption not used in production right now.
```

Comment:
```suggestion
One of the primitives we're using is the Legendre symbol, as a PRF. It would still be good to have it audited because it's an assumption not used in production right now.
```